### ISSUE TITLE
Update RTMP URL after running?

### ISSUE THREAD

**Issue by Junming-Chen** [TakuSemba/RtmpPublisher#22]
I wonder if there was a chance to change the rtmp url address after run the program
is there a way use other rtmp address, not just the coded in address?

**Comment by AlexUrrutia** [comment]
Yes, just create a text input field that has the RTMP path and a button to connect/disconnect

**Comment by namespace7** [comment]
> 
> 
> I wonder if there was a chance to change the rtmp url address after run the program
> is there a way use other rtmp address, not just the coded in address?

Take user URL input in EditText and pass it to URL Variable !

**Comment by vit1967** [comment]
Hepl me please^ How can write RTMP URL (what app/{stream-key}     need?)  for translate video from android (local ip1) to PC (local ip2) through WiFi (I can't connect to nginx-rtmp under Docker ) ?

**Comment by namespace7** [comment]
@vit1967  
You should be more clear about the Issue! what're you trying to do? what you have done till now !

**Comment by vit1967** [comment]
> @vit1967
> You should be more clear about the Issue! what're you trying to do? what you have done till now !

Im my case problem was successfully resolved by change...
private val url = BuildConfig.STREAMING_URL 
to
private val url = "rtmp://192.168.1.51:1935/live/s" //BuildConfig.STREAMING_URL  
in MainActivity.kt
Thanks for the good code)) But keep it simple ;)

**Comment by vit1967** [comment]
At the same time the question: I would also, in addition to the video, transmit "to the other side, through the nginx", also any messages. Is this possible using only nginx (and RTMP prot.)? Or do I have to hang a separate chat server for this?

**Comment by namespace7** [comment]
@vit1967 
you can also receive the video using ffmpeg library . write a command which will keep listening to a port number and play whenever there is a incoming stream.
